docs/Developers/Review.rst

```diff
@@ -140,6 +140,9 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check that the changes in the PR haven't been pushed from the master
+   branch.If it is so, then drop a comment saying that it is not a good
```

Comment:
Please make the following changes:
`branch.If` --> `branch. If`
`If it is so` --> `If this has happened`
`saying` --> `telling them`